internal/core/docker_host.go

```diff
@@ -66,10 +83,15 @@ func DefaultGatewayIP() (string, error) {
 //  4. Docker host from the default docker socket path, without the unix schema.
 //  5. Docker host from the "docker.host" property in the ~/.testcontainers.properties file.
 //  6. Rootless docker socket path.
-//  7. Else, the default Docker socket including schema will be returned.
+//  7. Else, because the Docker host is not set, it panics.
```

Comment:
question: Change the function name to MustExtractDockerHost to indicate this can panic?

Reply:
Makes total sense. I updated the name and deprecated the public API that retrieved the docker socket: See `testcontainers.go:ExtractDockerSocket`